Gemfile

```diff
@@ -48,3 +65,7 @@ group :development do
   gem 'spring'
 end
 
+group :test do
+  gem "capybara", "~> 2.7"
+  gem "launchy"
+end
```

Comment:
[Some comments](https://github.com/challengepost/the_commit/blob/master/Gemfile) of what gems are for would be nice, Im thinking of a junior coming in and not knowing anything about the gems.